### ISSUE TITLE
TITANS - COVID-19 X-Ray Detector - Deep Tech

### ISSUE THREAD

**Issue by chiragsamal** [skillenza-com/MishMash-India-2020#118]
**Before you start, please follow this format for your issue title**:
TITANS - COVID-19 X-Ray Detector - Deep Tech

## ℹ️ Project information

 1. **Theme** - Deep Tech / Machine Learning
 
2. **Project Name:** COVID-19 X-Ray Detection

3. **Short Project Description:** Website to detect and class

4. **Team Name:** Titans

5. **Team Members:** 
1) Deewanshu Ukey (https://github.com/deewan07)
2) Gourav Chowdhary (https://github.com/gouravcy)

6. **Demo Link:** Website (http://covid-19xraydetector.azurewebsites.net/)
YouTube Demo (https://youtu.be/xEhMQdvWVBs)

7. **Repository Link(s):** GitHub (https://github.com/chiragsamal/COVID19-Detection)

8. **Presentation Link:** https://docs.google.com/presentation/d/1kDHxq8bsFAtI52fLTnIr3CJpLGTQYxriw--z0_kOnUE/edit?usp=sharing

11. **Azure Services Used-** Azure WEB App

## 🔥 Your Pitch
The 2019 novel coronavirus (COVID-19), with a starting point in China, has spread rapidly among people living in other countries and is approaching approximately more than 800,000 cases worldwide according to the statistics of the European Centre for Disease Prevention and Control. There are a limited number of COVID-19 test kits available in hospitals due to the increasing cases daily. Therefore, it is necessary to implement an automatic detection system as a quick alternative diagnosis option to prevent COVID-19 spreading among people.

Example of an X-ray image taken from a patient with a positive test for COVID-19. Using X-ray images we can train a machine learning classifier to detect COVID-19 using Keras and TensorFlow. COVID-19 tests are currently hard to come by — there are simply not enough of them and they cannot be manufactured fast enough, which is causing panic.

Given that there are limited COVID-19 testing kits, we need to rely on other diagnosis measures.

For the purposes of this project, I thought to explore X-ray images as doctors frequently use X-rays and CT scans to diagnose pneumonia, lung inflammation, abscesses, and/or enlarged lymph nodes.

Since COVID-19 attacks the epithelial cells that line our respiratory tract, we can use X-rays to analyze the health of a patient’s lungs.

And given that nearly all hospitals have X-ray imaging machines, it could be possible to use X-rays to test for COVID-19 without the dedicated test kits.

A drawback is that X-ray analysis requires a radiology expert and takes significant time — which is precious when people are sick around the world. Therefore developing an automated analysis system is required to save medical professionals valuable time.

We have also build a web app to detect COVID-19 from chest X-rays of patients

## 🔦 Any other specific thing you want to highlight?
_(Optional)_
Advanced Deep Learning Methods are used for the segmentation and classification of COVID-19 Chest X-Ray Images.
The website for classification is built using Azure Services (WEB App).

## ✅ Checklist

**Before you post the issue**:
- [YES] You have followed the issue title format.
- [YES] You have mentioned the correct labels.
- [YES] You have provided all the information correctly.
